debian-setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Installs all of the necessary packages in a Docker container
```

Comment:
`set -eux`

at least `-x` so we can see what's going on in the logs.

Reply:
Sure.

Reply:
Addressed in 18515a3d90b68c06da043f1459e01e45670afe22